docs/connector-development/config-based/stream-slicers.md

```diff
@@ -0,0 +1,170 @@
+# Stream Slicers
+
+`StreamSlicer`s define how to partition a stream into a subset of records.
+
+It can be thought of as an iterator over the stream's data, where a `StreamSlice` is the retriever's unit of work.
+
+When a stream is read incrementally, a state message will be output by the connector after reading every slice, which enable checkpointing.
```

Comment:
You could readd a link to https://docs.airbyte.com/understanding-airbyte/airbyte-protocol/#state--checkpointing

Reply:
done